hail/src/main/scala/is/hail/utils/Interval.scala

```diff
@@ -226,4 +226,52 @@ object Interval {
       }
     }
   }
+
+  def sortAndReduce(xs: Array[Interval], o: ExtendedOrdering): Array[Interval] = {
```

Comment:
This looks equivalent to `RVDPartitioner.union(ktype, xs, allowedOverlap = ktype.size)`